src/components/Editor.tsx

```diff
@@ -1,6 +1,26 @@
 "use client";
-import MonacoEditor from "@monaco-editor/react";
+import MonacoEditor, { type Monaco } from "@monaco-editor/react";
+
+function configureEditor(monaco: Monaco) {
+  monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+    enableSchemaRequest: true,
+    validate: true,
```

Comment:
```suggestion
    validate: false,
```
It looks like we can't actually validate against this schema, just use it for completion:

[Example](http://localhost:3001/flows-ide?d=N4KABGBEDKAuCGsCmBnSAuMoIUgESQBslkMtwdcA5JAD1jMgEFDCBhAewFsAHY0gDQVKkACoBPHkkZMAxrACWHAHaQhlXG0SyAFmQDawjdg0ia9RgX5IAYvAXEAJmqOnIAJVQBXQrAAKiHqYkAAkAHRWJEgAogBOsRyxnig+DOqmOJBxCbHRAI5e8IRomIYZGszySsoAqsrwAEbEohzuXqrp5bhyiip2DkiO0bSySDy9Ha4ZlROiClxIHF4MU5QAuqtgAL5Ta52ZnFwLygzBNShIYKKx8MooAGZIsWCwHGCORFEvOpeOiPBge4JLjfS4KE5PBaOBSIS6yDisJBVFQuNwAdXssDmCzIAEYAOwAJgAHAAGUn7brI2qxQiMHSwWA8FDoAD0rPg1JQYQA5oQOA0vNzEjzWbAbndHrFWR9rKiKgBJZQ8ZYBWBBKDhHrVJUq2DcyLIGzApXIWJQmHIXXLeUiZKpNUa0IRT7Ie2+W2ZYajcbVADyym1fXshC8sWkmHuRQuRi2+zEEoeT1xZBMZjopyg11uSdihM9uAkUhk1ILUC0sF0Bk2aem5kzCZzUv6TjLdu8vkdjHC2clT2yiXdaU2uAHuQK0erXUyQdq9SaSBabUm06pExbg29YwmbYqs+xixtI7AG3KO1MeymkEOxwby9BYHBCkURReialYA4UhuE0BwIfKBLLEoxgPCiLUi8bzqmCELmoMlpwgixClpSUAYs+B54kSZIUles41LS9KMsybIclyvL8oKwqxKK4pNk8Yrvk8ZaQNa-iBN2YSzmx3K9rmuJsSxQ5dsEPZMbEuJDixW6+ioAazi2YYRm+XhILG8Z8VK+aYLW1AZpYrrSKhYiSMpMzVCxFZVqUNbHpA9aMJpTwbs4qHtiknYcaJYRObk8SDh2w7Tlk-njoUxRTqu5kqHUjTNK07S7iICkhpuIzbhZbl7tSB5LCs06nhk54aJebg3kgJyMPe0FgBc8LKI4b70c8X5PIg1R-twD7gmaFqwqBSFIr+rwPh8KCKPUv5gchO7GehWLzMpBIkuSxn4YRwQMkyLLspyEzcnyApCmEIqMc1Z19rELFsSJmpcdSPE+eJhKCcZwleXdvmElJxkyRM8nUop4ZkOKqnqUYzCsIcfBRKmV7RA1IOxKpxlFmZAQoGgxnlZVwRUIkXCvvCvD8NUAi1W84Y8IksC1V4sijJjYC3I1dDPixAQ3AsZolOQ5SQAsmPwDyZmiAA7m8cDC5cvkDSTsNTHGV7veqnE2PyYtSeDIiGrYqXODpV5o4wLaWfAQpmbrLnSaFjDQEglzjf14JgOEutjkOn73A+yPKJ+yx6gA3O8-xNZdYBi-AKB0wzqAPD4YSWdwt5VWMNMPk8OTMw1YAVY1NX3Brue0EiywWdrmT27A0BAaMSr3BwcNuA5wQADIcBwADWXg8AQ43gu1KhdqjpmMMMPDhpj1TRAAbkUSdHBVDZV+8JAhtHHDezVgFhiBfwIBz8Bc1EsS87pmQ78BSAAPr3AMYQALx27XSCOvXHBhB8CAOAaTCiEw+hSRrDCPUHEWVIAKBQNfcMsgwwoAUDPJAj9n671foEd+n814-wiH-ABQCwiQNvgiD4V1wGX1GEQwgJDkHBBrqgt+ygG6YO-sUMIPAOKKzeoFW6zo6FX3fpACuhZxLW0Nm4Y2wRTbY3NhcRyIj9Y2xyHbB2tUEDIEfH7MSzUPaBS9j7do-smQ2mxsnJeqdqaxFpjVTOiRs6sxzgXIudBS47iEVAduXce58NGLdc+9l9LBDYD8WQndvFoPVAIkexZgizjNpWDUZQMjn3TBYWhL9HQAFlIHwOUDyJKUAVZOnCGEx0Hju48B0R5IKXQQo5HyOFXmiTgr4XnPFZc+S1zVBcn9TKx5OkqFykeAqmxirrBMYvXG7iO7lIAi-MA7D1SQVXr1cEYJ9QvFMmAAAFHfYgMoFCxAAJRLI+LIBQHxHxb3ErVHQSwqFgAaJcGBcCEFGSvPNTCmBlo4TWtSAidJNrER2mRfaFEjrUVouJVkEVjKc3gNzJ4Z9NiQAWToMIIROLkKQai3cKLAjXxUIQcQSMUbIrztTHq19znosxS-AhzhOHK24R9Xh6T0GMMbm4mAbL1RZKnrkpuFQW5QBKYEPlOSeRSKNqPYI49J7wJULPeesKj7wpPki-mNjYg0KgAAcnFeCHkFNUHzMCGAXVYAADUxqr5sLNdai1KgBrgV-Ba61WL6WCNMErNwhS1Yay1pQH1Bxwywm4sqZYGqhUBKgBk+AnckCmkhPBWEeADksQkVAOVcdp5zzpOMlOtCSA926hG9ZDdnhFEIA+Wc0cRrfgrSCGqsAJaqOlmHXMh9j480FW4ONCa02xEEmImpqKdUQNgn1ZAdrFkOtZG6sA196rIHoGEX2VLXJ2XJRwSl1KMXeR6sm6EsJPUjPAb5ASZbe3TEII0Ig46RZqPFFs3VdFLq4mvrex5hBdXk11VLEWYAAwy2uRWsA6sOBizAPeMWz4dCPkagupdKgV2wDXe0Ddhz8mQDfbmKlyAuCNL6XzVcuBnmn1eTS7yYSMGEPI-AxBHSL4v2vmO-dd0PU4qym4MaE1B7KFY4Ecdh64LHunaiq15r52SeQxCVd67zmSd1dJ61NGOVhCxbfAYXrVyjIyKVGpmnt27scFRjjdLznYd4wPCY19jMnA3WZ50Imp1IMs2e5FX1h0kf5l++9T9giPsQM+19z1P13p-X+gDMtW2yzAxBqDMG4MIfNTJ5dGZ0MCfOVh7jUBcNSnw0gQjkVpzJOmPRyjAWRUv1o1AirjHcsiE02OqrE7eopvE-aqTSH0vyYw4pudC61NMM07st5pHcDWcmtUQT6onNu1QHxiYjpMH92myoWbehiN6YvLlyARmGoUoc3uziLmOtuc3cFKb-G7OHZ3cd0z7HnTXds9fU9Z5wFWxNJO873myu4GfEV7krX9BbLO2J7FXXlM9ZQxlhTjhDlrGw35wgD616MliC+2UUQqU-Yh+F79v7zW606iCJNomEKAlsQl6DBjYOLMUzDuTaH4c5bsvV5SoMkB7fs7ARzT3wjg4Qu9oqjLfXMtVt5cNeo0Bcqrn3JbOoOXXr0qkkVJAFc2eqMPaV0Ss20AnjmxVeaF6FvV7TcEjb+PMwaHlUai2tdOq41eOFCLT4q8yGzcawPGCEMNxcE4L7NfreUAwphX9144P-oAtYuq2f80IQ3KhTw0dPsx7q4P-Gw8fwj9gvAuCY8EKgUnkhRO7DFCQPH6YL2ZtjdT8F9PmflvsvD1g1h+fo-4NAUgIn0Oq88YdyHyh1DWtN+1y3nPbfuQ4rFxUP13kx8qAEXLjXg+s8fT8cK68obkDS8jRmmV+v-cKuUEq-NV4cYNmhssS4i+-aOkBD4at41YiGoeVHQYn4-ZYrsavNb1uJN+RPEeANEG4u01Ue0R0B9-9bMWtGAtka8Ns3sJNBtJNhsP5RsBhjkFBvYtk79ath8nhPxngtktllAOBaZ8D1Mvd9RjkWYwB0Ci9oFS4KNEFDljkiALg-9FckCZ0ttvUuEqkeEFsYDx9VYuUgkkRQkeUdABEoDMhM1rxbkFBRhLJlDGYStjA7It9RVeVslDVsMAA1I+GEBcTiXQ2Q9TXPdvAvIBbDBUFAPwSeMxSMaMNSM8XYAtFwqAAgFZZQS4ZtTZTeWZE1Z3NwAgKMVIZRauGrZXLlAAcRIDCU4BdQsnkNVyv2CWkNQRSJmh1EuwqEUNiS8MmUgESItw5XNGt0aDt23jmVyKGl6XRExE+TAG+VWjwj+Q2igC2hIl2nIkOiohOhogulzFZGmkaI20t0bhVW7URQ91wF5351pVQRFyDUEIdBZWKRfmiB4GX3WIhn7UTTxwQkHQ938TV0bHfQPz12ihXAqEv0YDYB30uABAIxAOhBgVeFiHEC-1LXawhyWVuXuRqn3ngBYg+UWiwhWlwjcHWgBR6KBVIj2mqAOkomOlOnywYi4E7k+OujLWEPugmEeiOMHVeiZSEK2LCAyVxIOUDRwGDWoA4DjXqBFkcAaK5HOK33xhZOlnZMGkBhDBuLMmzSnmN2VRd1VTdyjTcC1XHV1SYFpmICjlphUEuGCJqgmK5FS3dQs0agdWIJ1O4MdwEzewG1Sy4CFFpkeTAEJlZMGDCF1R0wOPF0pMlzugS3pIgEZKmWALv18SvB0NX1EKX2VyiTMmKIv20ASVsmCiDNgDvwNVyUiTsnnzugsLKR7kqSiL2zHHqUnBslIzuNigXCXESkaygBSgGCGHSlknuOaRykWjymdP0xGU8Iv1MVKMzJANBLX1-AkxGi-khFWUfHWVgE2R2QGH2SOQhJaKhK+Wwg6LhK6IRMgF6OBRRJUDRPBWGMhXOhhUlLmPd3SMyDY04kQNNL4NxVRQJWUCJRJW5zJTuxM3m1Wx4MvPcwEIpM2PdOdH9InxbJ9O5XoTFX0NyVNhPJMluKlTKhkTMgsKTLyV+ltiLUdjURgldjCE9N0Q1J+DAF9kMT1FN28I8DTksQzlCl-zzgfELkg2LhcXLhdIOCyOSIFOJINh8wUMPyUJ3VUOxnUNQE0MoH+ygD9CukLNXBEuSkRgkom04rksgGMNfzijMm2NQV2IwWsO5BSAaBQFkFfzrI3SYzcAcKcNQG8K52IyYrkqkumAcKoCfzICjArwrLcCUtMOIHMJ2L2KsKnw0y8F0v0oUEMs-Lku2CspPG2wrNsu6BkrACaQmxioqHcpUq8vUp8tbxYW5CXmfHEGvnHOLFcsVEcOcMmUsrCu2CKogCSpEBSrMOo28s0r8pyvHPytHiqsrnFENXzIimCHiLYD8Gvh5FUnGmQ1SJRAiu9IisKmnB2xKj2y327IsJbKDXbLKk7MyKkO+EQBCKvmdTyKdUgVtNuD5JYgiPNg9D6pIHJ1cw5NcWsqgB5JOrZLuq6SFMgsUJgoeLgsYCevtP5PGvrK9BQvVzQudk0SwoDRwq3jwoIryiIpKLvDIqsTwq1SoocTwtoqg2cVgXuoZPjHKLv1epREgq30kJCSJrYvyOFJLFmg7ImQbHKNAIJmqNt2WHtxDL9mJqBtwEhJxAXJhN+QmH+SIm2mRIGPRIhVGKlHGKpqmLDMPIgPmJPMWOfIe1fIvOvjNIZS-NdJ-KKRdE5o0riIerKOupONhG5q5JjSUKkJuvO25oVAKJECKJQnprNzNoqKt1-BqPZpqiF36itrmjnP5raMXNhOymFu6LXKRP6NBUGIxJGKxOlC1Ns2mPAOlIWKgCWJOwPQtunVCoetwDTOdHtoh2NrAJXwTL7OqEQrkPkseptvlxrqHg4nDMYAxixndpIpXmsQNyGk-wvOZqqN-BwIfCAJmSjB-gzvVSzsgBoN5mcouHAUT2IWYlcIr1n3cn1s4ioMrtNvJs7kpsBqdvOMUKCV4vGweIEqI3KBEsgDEqEqSQiuYDioStIxqsyDqs8oXxbtPwysnyyv8sCoMrTudrCtYhKvMrKuRncLCp9I-smsyHssco3uXqQdwG-tUsNvfIrsAcjx0r0tAZm0LomzmvKBmq6AQefoUqYDfqQc-swZMNSt-qNoAeYUjxaryoKqvoqsgbMoD0zHKvgY6sYagCwd3r-rwY4ewS4basKowZFS6tyR6t5jKIGqGpGr51TsYvgemu22PAM35kWumS8RkJWoZLWoeI2qeKyO2pRt+D-v2smL9iOrtNOuMnOqiLxmZOesGG5tlwPqyLLoQkdo4r8XPoErUMvtvpoZqUftkq6CStfo4vfsksUcUuYfqruhCdhGka0uAaIeCrAeMuKoEYstgaQeodKwyZQdYCcrcI6qYeUuydLvzpiHYYKcIaCpCp1r0bksoY+2I2SbodSYYYyYke8lyeQHyeapOFyvkd4YgdMtKqEcqYquqaSYmayZ-pyfadmaAbkZ4dKZEDgFfxUYnF6qgH6sGuGsWzGoOtUCqf0dI3IZwCMbrBtqWvMbbN227tKMPvsf+KPUpx0cOujncbZLOqQEiMuuueDPfKttjAoGDWvBseCDi3-ABB-zBb9hGgBCHtxbACFDf1uA0QBJhB+OcbdtwClksUVMYHKNYsBsES2CAA)

<img width="1457" alt="Screenshot 2024-04-10 at 4 25 25 PM" src="https://github.com/globus/flows-ide/assets/694253/bc4b08b1-711e-464e-918b-aba92a7c01bd">

That is the [Two Stage Globus Transfer definition](https://app.globus.org/flows/99791f7d-6c2c-4675-af4b-b927db68bad0/definition), so it is valid service-side.

Reply:
Huh that's surprising

Reply:
Okay, I'll work on fixing up that schema, I'm guessing we just need some `"additionalProperties": true` in the right places

Reply:
Ideally we don't augment that schema (in this repository), from what I understand it is auto-generated; See https://github.com/globus/flows-ide/pull/30#discussion_r1560085713